include/llvm/IR/IntrinsicsMemCap.td

```diff
@@ -123,6 +123,10 @@ def int_cheri_cap_diff:
 ////////////////////////////////////////////////////////////////////////////////
 // Pointer / capability interoperation intrinsics
 ////////////////////////////////////////////////////////////////////////////////
+def int_cheri_cap_address_get:
+  GCCBuiltinAlias<"__builtin_mips_cheri_cap_address_get">,
```

Comment:
Don't add a legacy compat alias.  We don't need to maintain compatibility with legacy code for a new alias.